Otherwise you end up with a leading double-slash, which is reserved
for UNC paths on Cygwin. (similar to old mkdirhier bug)

--- origsrc/zephyr-3.0/ensure-dir.sh	2007-12-19 19:17:23.000000000 -0600
+++ src/zephyr-3.0/ensure-dir.sh	2010-11-02 12:10:52.015728600 -0500
@@ -5,7 +5,7 @@ mode=$2
 OLDIFS="${IFS}"; IFS=/; set $pathname; IFS="${OLDIFS}"
 
 case $pathname in
-	/*)	partial=/; ;;
+	/*)	partial=; ;;
 	*)	partial=""; ;;
 esac
 
